Makefile

```diff
@@ -58,8 +58,22 @@ rust-build-mock-token:
 
 rust-build-near: rust-build-omni-bridge rust-build-omni-token rust-build-token-deployer rust-build-omni-prover rust-build-evm-prover rust-build-wormhole-omni-prover-proxy rust-build-mock-prover rust-build-mock-token
 
-rust-build-solana:
-	cd solana/bridge_token_factory && RUSTUP_TOOLCHAIN="nightly-2024-11-19" anchor build --verifiable
+solana-generate-program-id:
```

Comment:
@kiseln  the `rust-build-solana` was used in this [workflow](https://github.com/Near-One/omni-bridge/blob/533f2fde3ee59c0d4c2bbde2e8797fda4b0293ce/.github/workflows/update-contracts.yaml#L33), can you please fix it.

Reply:
9f38bfda2255921c977dd6b60aabce127d72da08

Reply:
I think this will not work because there is no key file.

Reply:
All checks have passed. I don't think we need a key file for building

Reply:
It passed because the build release is trigger only on creating a tags